src/main/java/seedu/address/model/course/UniqueCourseList.java

```diff
@@ -137,10 +137,10 @@ public String toString() {
     /**
      * Returns true if {@code persons} contains only unique persons.
      */
-    private boolean personsAreUnique(List<Course> Courses) {
-        for (int i = 0; i < Courses.size() - 1; i++) {
-            for (int j = i + 1; j < Courses.size(); j++) {
-                if (Courses.get(i).isSameCourse(Courses.get(j))) {
+    private boolean personsAreUnique(List<Course> courses) {
```

Comment:
Sharp eyes! Good that you follow cs2103t naming convention closely. Good job phinehas faithful tan!